docs/community/KPI-options.md

```diff
@@ -0,0 +1,41 @@
+---
+title: KPI Options
+sidebar_label: KPI Options
+---
+## KPI Option Overview
+
+Community options, growth options or performance options - the naming is not the point, more that these options are designed to create aligned incentives with your community to grow. We have decided to call them *Key Performance Indicator options* or *KPI options*. 
+
+### What are these options? 
+
+Using UMA’s Expiring Multi Party (EMP) contract template, any project can create an incentive mechanism which offers rewards to their targeted community in exchange for the community’s support in helping them achieve their Key Performance Metrics. 
+
+At a high-level, KPI options are synthetic ERC20 tokens that can be redeemed , but only if the project hits it’s predetermined KPI targets before expiry. These targets are up to each project and could be TVL, number of contracts deployed, number of transactions, etc.
+
```

Comment:
KPI options  are ERC20 tokens which can be redeemed for a pre-determined number of protocol tokens, but only if the project reaches its predetemined KPI targets before the expiry date.  These targets can include the Total Value Locked, number of transactions, percentage of governance participation, token price....or any other metric you care to set. 

TVL options are a SMART choice

- Specific - you define the metic in advance
 


- advance